api/src/main/java/org/eclipse/microprofile/lra/client/LRAClient.java

```diff
@@ -379,4 +277,22 @@ URL updateCompensator(URL recoveryUrl, Class<?> resourceClass, URI baseUri, Stri
      * @return the current LRA (can be null)
      */
     URL getCurrent();
+
+    /**
+     * Lookup the status of an LRA
+     *
+     * @param lraId the LRA whose status is being requested
+     *
+     * @return the status of the the LRA.
+     *
+     * @throws NotFoundException if the LRA no longer exists
+     *
+     * @throws UnknowableException if the status is unknowable
+     *
+     * @throws GenericLRAException if the request to the coordinator failed.
+     * {@link GenericLRAException#getCause()} and/or
+     * {@link GenericLRAException#getStatusCode()}
+     * may provide a more specific reason.
+     */
+    ParticipantStatus getStatus(URL lraId) throws GenericLRAException;
```

Comment:
It seems to me inconsistent to return a specific participantstatus from a getStatus method that describes itself capable to lookup an LRA and return the status of said item.